.github/ISSUE_TEMPLATE.md

```diff
@@ -1,3 +1,4 @@
+<!-- Keep the progress checklist.  -->
 ## Progress
```

Comment:
I like it, maybe move it to the bottom so it isn't the first thing someone sees.